cfg/cis-1.5/master.yaml

```diff
@@ -297,8 +303,22 @@ groups:
 
       - id: 1.1.20
         text: "Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Scored)"
-        audit: "stat -c %n\ %a /etc/kubernetes/pki/*.crt"
-        type: "manual"
+        audit: |
```

Comment:
I think It will better to use bitmask flag here and  multiple values.
It will be more accurate